src/space_stage/SpaceFleet.cs

```diff
@@ -5,13 +5,15 @@
 using Nito.Collections;
 
 /// <summary>
-///   A fleet (or just one) ship out in space.
+///   A fleet of ships (or just one ship) out in space.
 /// </summary>
 public partial class SpaceFleet : Node3D, IEntityWithNameLabel, IStrategicUnit
 {
     [Export]
     public NodePath? VisualsParentPath;
 
+    // public Node _engineEmittor;
```

Comment:
I think this commented out code in this class (also below in this file) should be removed if it is not going to be implemented.

Reply:
Yes. That part could be done later.

Reply:
Then I'd say that this commented out code definitely needs to go, especially as the property here is named against our code naming conventions so uncommenting it would cause an immediate warning.